Vida de los Goliath pero bien hecho

rtlui opened the pull request:
Este pr no es mio, solo lo traje de los pr rechazados de paradise y lo modifique y adapte a nuestro code actual.
Link al pr original: https://github.com/ParadiseSS13/Paradise/pull/13075

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
Añade la posibilidad de domesticar goliaths, mejorarlos y su crecimiento desde bebes hasta adultos, sumado a que el ancient goliath ahora es mas dificil de vencer ya que tiene una mecanica especial.
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Le añade profundidad a lavaland y es un implemento bastante interesante para esos mineros que quieren usar su tiempo para otras cosas que no sea matar jefes indiscriminadamente o simplemente se sientan solos autisteando en lavaland.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Añade los goliaths juveniles y subadultos
add: Sistema para domesticar goliaths
add: Goliaths controlados por jugadores
add: Un nuevo tipo de goliath volcanico
add: Nueva habilidad para el Ancient Goliath
tweak: Los goliaths ahora regeneran vida MUY lentamente
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

rtlui committed:
TODO JUNTITO

alfin ctm

Dapple27 commented:
Otro pr para la misma vaina? muh mal Patata

rtlui committed:
dream maker maricon deja de llorar por un null

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -45,30 +102,33 @@
 
-/mob/living/simple_animal/hostile/asteroid/goliath/revive()
-	..()
-	anchored = TRUE
```
por qué eliminas esto de paradise? devuelvelo.

rtlui replied:
evan esta en paradise.....

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -89,7 +149,7 @@
 	pre_attack_icon = "goliath2"
 	crusher_loot = /obj/item/crusher_trophy/goliath_tentacle
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/goliath= 2, /obj/item/stack/sheet/animalhide/goliath_hide = 1, /obj/item/stack/sheet/bone = 2)
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/goliath = 2, /obj/item/stack/sheet/animalhide/goliath_hide = 1, /obj/item/stack/sheet/bone = 2)
```
si no vas a tocar esto para nada no ponga este espacio acá.

rtlui replied:
EVAN QUE PUSE UN LOOT EXTRA

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -113,12 +175,14 @@
 	throw_message = "does nothing to the rocky hide of the"
 	loot = list(/obj/item/stack/sheet/animalhide/goliath_hide) //A throwback to the asteroid days
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/goliath= 2, /obj/item/stack/sheet/bone = 2)
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/goliath = 2, /obj/item/stack/sheet/bone = 2)
```
si no modificas esto para otra cosa tampoco pongas un espacio. solo dará conflictos.

rtlui replied:
EVAN QUE HAY LOOT EXTRA x2

Evankhell561 replied:
no le des a resolver caundo no has hecho lo que pedi

SagaStars committed:
Control civil

Hispania no perdera este enfrentamiento, la primera linea frontal no puede retroceder.

rtlui committed:
Update goliath.dm

rtlui committed:
Merge branch 'GOLIATHS' of https://github.com/PotatoTheDeveloper/Paradise into GOLIATHS

rtlui committed:
Revert "Update goliath.dm"

This reverts commit 5f0c8e6515eeb36bc23c6c2badf4c7ddbb469946.

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -98,7 +158,9 @@
-		return INITIALIZE_HINT_QDEL
+	else if(prob(10))
+		new /mob/living/simple_animal/hostile/asteroid/goliath/juvenile(loc)
+	return INITIALIZE_HINT_QDEL
```
esto hará que el goliath normal nunca se genere. Es decir que o tienes un anciano (con una prob de 1) o tienes un juvenil con una prob de 10 pero jamas tendrás un goliath normalito. y eso es un problema.

rtlui committed:
YA? TE GUSTA? QUERES UNA CHUPADA TAMBIEN?

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+	loot = list()
+
+/mob/living/simple_animal/hostile/asteroid/goliath/juvenile/subadult
+	growth = 599
```
pon esto en seiscientos y aumentale la vida con respecto al juvenil en 150 extra

Evankhell561 replied:
el sub adult deberia tener speed 2

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -195,3 +278,26 @@
 	deltimer(timerid)
 	timerid = QDEL_IN(src, 7)
+
+/mob/living/simple_animal/hostile/asteroid/goliath/death()
```
ya existe el proc death para los goliaths en paradise, fusiona el otro con el tuyo.

rtlui committed:
cambios

rtlui committed:
uy casi buffeo a los adultos

rtlui committed:
flash! esto nunca paso

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -102,11 +102,17 @@
 	move_resist = MOVE_RESIST_DEFAULT
 	pull_force = PULL_FORCE_DEFAULT
 	..(gibbed)
+	add_draconian_effect(draconian_overlay)
```
pon esto antes del ..(gibbed)

rtlui committed:
wa

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+	var/picking_candidates = FALSE
+	var/food_wanted = MEAT // Meat by default.
+	var/feed_cooldown = 0
+	var/draconian = NOT_DRACONIAN // Can be NOT_DRACONIAN, DRACONIAN, FULL_DRACONIAN.
```
no veo que esto cambie en ningun momento, por qué lo añades?

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+#define DIAMOND /obj/item/stack/ore/diamond
+#define CYBERORGAN /obj/item/organ/internal/cyberimp
+#define DRAGONSBLOOD /obj/item/dragons_blood
+#define GROWTH_MAX 1200
```
envia a estos defines a su propio archivo para que no tengas que repetirlos, tambien deberias quitar el undefine.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+		..()
+
+/mob/living/simple_animal/hostile/asteroid/goliath/Destroy() // When gibbed / deleted, the ancient goliath that spawned it will be able to spawn another.
+	..()
```
el proc destroy siempre debe retornar algo, ponle . = ..() en vez de ..(), tambien, llamar al pariente debe ser lo ultimo que hagas al usar el proc destroy

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -48,27 +105,36 @@
 /mob/living/simple_animal/hostile/asteroid/goliath/adjustHealth(amount, updating_health = TRUE)
-	ranged_cooldown -= 10
-	handle_preattack()
+	if(amount > 0)
```
why?

rtlui replied:
para que las heal no lo activen

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+	gender = NEUTER
+	environment_smash = 1
+	sentience_type = SENTIENCE_OTHER
+	loot = list()
```
esta lista debe irse llenando a medida que el goliath crece.

Evankhell561 replied:
según veo el beast no tiene loot por lo que esto seria solo una variable repetida. en general elimina todos los loot = list() de tu code.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+/mob/living/simple_animal/hostile/asteroid/goliath/proc/add_draconian_effect() // Apply the draconian overlay
+	cut_overlay(draconian_overlay)
+	var/status
+	if(draconian == NOT_DRACONIAN)
```
draconian segun veo en tu code siempre sera igual a not dragonian asi que nunca hará nada

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+
+/mob/living/simple_animal/hostile/asteroid/goliath/proc/add_draconian_effect() // Apply the draconian overlay
+	cut_overlay(draconian_overlay)
+	var/status
```
declara esta variable en la linea 263, justo antes de que vayas a usarla, no acá.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,358 @@
+	var/draconian = NOT_DRACONIAN // Can be NOT_DRACONIAN, DRACONIAN, FULL_DRACONIAN.
+	var/mutable_appearance/draconian_overlay
+	var/aux_tentacles = 3 // Auxillary tentacles. The total amount of tentacles is 1 + [aux_tentacles] + [extra_tentacles].
+	var/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient/leader = null
```
no inicialices variables como null

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -177,13 +262,18 @@
 		visible_message("<span class='danger'>[src] grabs hold of [L]!</span>")
-		L.Stun(5)
-		L.adjustBruteLoss(rand(10,15))
+		if(istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT)
```
```suggestion
		if((istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT) || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
```
si el spawner no es un goliath entonces deberia mantener el efecto anterior.

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -177,13 +262,18 @@
 	timerid = addtimer(CALLBACK(src, .proc/trip), 3, TIMER_STOPPABLE)
 
 /obj/effect/temp_visual/goliath_tentacle/proc/trip()
+	var/mob/living/simple_animal/hostile/asteroid/goliath/G = spawner
```
no entinedo la necesidad de declarar G cuando en la linea de abajo pudiste hacer. esto:
if(istype(spawner, /mob/living/simple_animal/hostile/asteroid/goliath) 
la variable G me parece inutil.

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -30,13 +49,51 @@
+			Goto(pick(surrounding_turfs), move_to_delay)
+			return
+
+/mob/living/simple_animal/hostile/asteroid/goliath/revive()
```
hay dos procs llamados revive, elimina este que es una copia del otor.

rtlui replied:
ya elimine 1 listardo

rtlui committed:
Merge remote-tracking branch 'upstream/Lambda' into GOLIATHS

rtlui committed:
fix numero 1283913813812831

poco a poco descendiendo a la locura

rtlui committed:
linea DE MIERDAAAAA

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+//goliath taming code
+/mob/living/simple_animal/hostile/asteroid/goliath
+	var/growth = 1200 // Out of 1200.
```
```suggestion
	var/growth = GROWTH_MAX //1200.
```

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+	var/list/ghost_volunteers[0]
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/Life()
+	. = ..()
```
lo primero que deberias hacer acá luego del . = ..() es comprobar si el goliath esá vivo o no y si no lo está, salir de este proc con un return FALSE puesto que segun veo nada en tu code tiene sentido en caso de que esté vivo. si me equivoco me avisas.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+	add_draconian_effect(draconian_overlay)
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/death(gibbed)
+	..(gibbed)
```
el ..() va de ultimo

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -56,19 +58,26 @@
 	if(get_dist(src, target) <= 7)//Screen range check, so you can't get tentacle'd offscreen
-		visible_message("<span class='warning'>[src] digs its tentacles under [target]!</span>")
-		new /obj/effect/temp_visual/goliath_tentacle/original(tturf, src)
+		if((aux_tentacles != 0) || (growth_stage == JUVENILE && draconian != NOT_DRACONIAN))
```
lo primero que deberias hacer en este proc es ver si vas a generar los tentaculos extra y fijar cuantos serán, luego poner algo como esto
`visible_message("<span class='warning'>[src] digs [aux_tentacles + extra_tentacles  > 0 ? "its tentacles " : "a tentacle"] under [target]!</span>")`
otra cosa es que tampoco entiendo por qué si cada status del goliath tiene una cantidad de tencaculos extra distintos para cada uno, necesitas añadir otro en caso de que el goliath sea un anciano. bastaria con cambair la var aux_tentacles  para cada nivel de crecimiento.

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -56,19 +58,26 @@
 	..(gibbed)
 
 /mob/living/simple_animal/hostile/asteroid/goliath/OpenFire()
+	var/extra_tentacles = 0
```
deberias declarar esta variable luego del primer return debido a que antes de pasar el primer return de este proc es inutil esta var, asi como que tampoco entineod para que la nitas si cada nivel de crecimiento tiene una cantidad de tentaculs extra distintos.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+		environment_smash = 2
+		aux_tentacles = 1
+		add_draconian_effect()
+	if(growth >= 1200 && growth_stage == SUBADULT && !stat) // Grow to adult
```
GROWTH_MAX

Evankhell561 replied:
le diste a resolver esto pero no parece estar resuelto.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+			user.drop_item()
+			qdel(O)
+			playsound(get_turf(src), 'sound/items/drink.ogg', 50, 0)
+			tame_progress += rand(950, 1200)
```
GROWTH_MAX

rtlui replied:
tame_progress no tiene nada que ver con el crecimiento

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+		visible_message("<span class='warning'>[src] looks frenzied!</span>")
+		food_wanted = MEAT
+		faction = list("mining")
+	if(tame_progress >= 600 && tame_stage == WILD) // Become neutral
```
veo que 600 es una var importante en tu codigo, haz un define con  él.

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+			handle_tame_progress()
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/proc/handle_tame_progress()
+	if(tame_progress <= 599 && tame_stage != WILD) // Become feral if left alone for too long
```
esto esta bien pero es mejor colocar tame_progress < 600

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,366 @@
+			visible_message("<span class='notice'>\The [src] looks undecided...</span>")
+			tame_progress = 2100
+			picking_candidates = FALSE
+	return
```
este return sobra si no retornarás nada mejor no ponerlo.

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -30,6 +30,8 @@
 	move_force = MOVE_FORCE_VERY_STRONG
 	move_resist = MOVE_FORCE_VERY_STRONG
 	pull_force = MOVE_FORCE_VERY_STRONG
+	gender = NEUTER
```
por qué le añades esto?

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -30,6 +30,8 @@
 	move_resist = MOVE_FORCE_VERY_STRONG
 	pull_force = MOVE_FORCE_VERY_STRONG
+	gender = NEUTER
+	sentience_type = SENTIENCE_OTHER
```
why?

rtlui committed:
a

rtlui committed:
Update code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm

Co-authored-by: Evankhell561 <49106866+Evankhell561@users.noreply.github.com>

rtlui committed:
e

rtlui committed:
Merge branch 'GOLIATHS' of https://github.com/PotatoTheDeveloper/Paradise into GOLIATHS

rtlui committed:
Update goliath.dm

rtlui committed:
Update goliath.dm

rtlui committed:
Update goliath.dm

rtlui committed:
Update goliath.dm

rtlui committed:
Update goliath.dm

rtlui committed:
Update goliath.dm

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,368 @@
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/monstermeat/goliath = 1)
+
+/mob/living/simple_animal/hostile/asteroid/goliath/beast/juvenile/subadult
+	growth = 599
```
```suggestion
	growth = GROWTH_HALF-1
```

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,368 @@
+	move_force = MOVE_FORCE_DEFAULT
+	move_resist = MOVE_RESIST_DEFAULT
+	pull_force = PULL_FORCE_DEFAULT
+	gender = NEUTER
```
por qué haces a los goliaths de genero neutro?

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,368 @@
+				to_chat(user, "<span class='warning'>\The [src] is dead!</span>")
+			else if(feed_cooldown >= 1)
+				to_chat(user, "<span class='warning'>\The [src] is not hungry yet!</span>")
+	else if(istype(O, DRAGONSBLOOD) && draconian <= 1)
```
```suggestion
	else if(istype(O, DRAGONSBLOOD) && draconian <= DRACONIAN)
```

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,368 @@
+				feed_cooldown--
+			handle_tame_progress()
+			handle_growth()
+		if((stat == DEAD) && (tame_stage != TAMED)) // So that 1) an ancient goliath can't immediately spawn another juvenile goliath if it dies, and 2) can't easily cheese the juvenile goliath taming process
```
que quiere decir este comentario?

rtlui replied:
el ancient goliath puede spawnear 3 goliaths juveniles, o hacer sus subditos los goliaths en su vision con un maximo de 3, es y el numero de subditos se reduce cuando el goliath desaparece no cuando muere, y ese codigo es para que el ancient no spamee el spawn basicamente, detiene que spawnee constantemente juveniles

Evankhell561 commented on code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -0,0 +1,368 @@
+/mob/living/simple_animal/hostile/asteroid/goliath
+	var/growth = GROWTH_MAX //1200.
+	var/growth_stage = ADULT // Can be ANCIENT, ADULT, SUBADULT, JUVENILE.
+	var/tame_progress = 0
```
este tammet progresss... al final lo has codeado tu, deberias especificar en algun lado cual es el valor maximo de esta variable, en algun define. y hacer el mismo tabajo que has hecho con la var growth.

rtlui replied:
tame_progress no tiene maximo pero el codigo hace una cosa rara y cuando fallas el tameo final te lo reduce a 2100 pero tecnicamente puedes subirlo hasta 3000, a los 2600 basicamente el bicho se tamea, si esto falla te devuelve a 2100, si no puedes seguir aumentandolo aunque deberia poder hacer el tope 3000

rtlui replied:
Resumen: no tiene maximo

rtlui committed:
Update code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm

Co-authored-by: Evankhell561 <49106866+Evankhell561@users.noreply.github.com>

rtlui committed:
Update code/hispania/modules/mob/living/simple_animal/hostile/mining/goliath.dm

Co-authored-by: Evankhell561 <49106866+Evankhell561@users.noreply.github.com>

rtlui committed:
Update goliath.dm

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -60,15 +60,16 @@
-		visible_message("<span class='warning'>[src] digs its tentacles under [target]!</span>")
-		new /obj/effect/temp_visual/goliath_tentacle/original(tturf, src)
+		visible_message("<span class='warning'>[src] digs [aux_tentacles > 0 ? "its tentacles " : "a tentacle"] under [target]!</span>")
+		new /obj/effect/temp_visual/goliath_tentacle/original(tturf, src, (aux_tentacles))
```
```suggestion
		new /obj/effect/temp_visual/goliath_tentacle/original(tturf, src, aux_tentacles)
```

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -137,6 +144,20 @@
+		for(var/mob/living/simple_animal/hostile/asteroid/goliath/S in range(7,src))
+			if((S.leader == null || S.leader.stat == DEAD) && S.tame_stage == WILD && (S.growth_stage != ADULT && S.growth_stage != ANCIENT))
+				S.leader = src
+				goliaths_owned++
```
eso esta bien pero me encantaria que metas todo esto dentro de un proc y lo pongas en hispania.
```suggestion
	handle_goliaths_owner()
```

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -177,13 +198,18 @@
 		visible_message("<span class='danger'>[src] grabs hold of [L]!</span>")
-		L.Stun(5)
-		L.adjustBruteLoss(rand(10,15))
+		if(istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
```
```suggestion
		if((istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT) || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
```

rtlui committed:
OLE OLE OLE OLE

OLEEEEE OLEEEEEEEEE

rtlui committed:
IM VERY LOCO

rtlui committed:
Update goliath.dm

rtlui committed:
nunca me cansare de cagarme en la linea de mierda

rtlui committed:
bot de mierda deja de tirar cosas que no tienen que ver conmigo

rtlui committed:
flashaste confianza

Evankhell561 commented on code/modules/mob/living/simple_animal/hostile/mining/goliath.dm:
```diff
@@ -191,7 +191,7 @@
 			continue
 		visible_message("<span class='danger'>[src] grabs hold of [L]!</span>")
-		if((istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
+		if(istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
```
```suggestion
		if((istype(G, /mob/living/simple_animal/hostile/asteroid/goliath) && G.growth_stage == ADULT) || !istype(G, /mob/living/simple_animal/hostile/asteroid/goliath))
```

rtlui committed:
Update code/modules/mob/living/simple_animal/hostile/mining/goliath.dm

Co-authored-by: Evankhell561 <49106866+Evankhell561@users.noreply.github.com>
